src/com_tjvendors/admin/models/payout.php

```diff
@@ -227,12 +229,13 @@ public function save($data)
 			$payout_update->id = $id;
 			$payout_update->transaction_id = $data['transaction_id'] . $payout_update->id;
 
+			// Plugin trigger
+			PluginHelper::importPlugin('tjvendors');
+			Factory::getApplication()->triggerEvent('tjVendorOnAfterPayoutPaid', array($id, $data, true));
```

Comment:
dont triggers needs to be starting as `on`

Reply:
done